lib/between_meals/repo/git.rb

```diff
@@ -168,7 +173,7 @@ def parse_status(changes)
         # X: "unknown" change type (most probably a bug, please report it)
 
         # rubocop:disable MultilineBlockChain
-        changes.lines.map do |line|
+        changes.lines.to_a.reverse.map do |line|
```

Comment:
Can you provide an example git output? And incorporate that into the test cases? I've got a feeling this fix is incidental and possibly breaking other stuff.